src/DssSpell.sol

```diff
@@ -33,8 +44,18 @@ contract SpellAction {
         ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
 
     // Ilks
-    bytes32 constant ILK_ETH_A          = "ETH-A";
-    bytes32 constant ILK_WBTC_A         = "WBTC-A";
+    bytes32 constant ILK_LINK_A         = "LINK-A";
+    bytes32 constant ILK_MANA_A         = "MANA-A";
+    bytes32 constant ILK_BAT_A          = "BAT-A";
+    bytes32 constant ILK_TUSD_A         = "TUSD-A";
+    bytes32 constant ILK_PSM_USDC_A     = "PSM-USDC-A";
+
+    // Lerp Module
+    address constant LERP_FAB = 0x9B98aF142993877BEF8FC5cA514fD8A18E8f8Ed6;
```

Comment:
Could you put a comment linking to this repo here?

Reply:
Added to the PR description.